nixos/modules/services/networking/dae.nix

```diff
@@ -0,0 +1,42 @@
+{ config, pkgs, lib, ... }:
+
+{
+  options = {
+    services.dae = {
+      enable = lib.options.mkEnableOption (lib.mdDoc "the dae service");
+    };
+  };
+
+  config = lib.mkIf config.services.dae.enable {
+    environment.systemPackages = [ pkgs.dae ];
+
+    networking.firewall.allowedTCPPorts = [ 12345 ];
+    networking.firewall.allowedUDPPorts = [ 12345 ];
+
+    systemd.services.dae = {
+      unitConfig = {
+        Description = "dae Service";
+        Documentation = "https://github.com/daeuniverse/dae";
+        After = [ "network.target" "systemd-sysctl.service" ];
+        Wants = [ "network.target" ];
+      };
+
+      serviceConfig = {
+        User = "root";
+        ExecStartPre = "${lib.getExe pkgs.dae} validate -c /etc/dae/config.dae";
+        ExecStart = "${
+            lib.getExe pkgs.dae
+          } run --disable-timestamp -c /etc/dae/config.dae";
+        ExecReload = "${lib.getExe pkgs.dae} reload $MAINPID";
+        LimitNPROC = 512;
+        LimitNOFILE = 1048576;
+        Restart = "on-abnormal";
+        Type = "notify";
+      };
+
+      wantedBy = [ "multi-user.target" ];
+    };
+  };
+
+  meta.maintainers = with lib.maintainers; [ pokon548 ];
```

Comment:
Usually we put this as the first  attribute, in order to quickly find the culprits.

Reply:
Thanks!